src/workload.py

```diff
@@ -40,20 +44,30 @@ def start(self) -> None:
             logger.exception(str(e))
 
     @retry(stop=stop_after_attempt(3), wait=wait_fixed(5), reraise=True)
```

Comment:
```suggestion
    @retry(stop=stop_after_attempt(3), wait=wait_fixed(5), reraise=True, retry=tenacity.retry_if_exception_type(EtcdServiceError))
```
so that install is not retried for other exceptions when called in refresh_snap